### ISSUE TITLE
strange names of loaded native libraries

### ISSUE THREAD

**Issue by judovana** [netty/netty-tcnative#23]
hi! I'm wondering why netty-tcnative is laoding normal tomcat libraries, and not the library it is building.

The patch to fix it is:
```
# this patch is suspicious
# why netty-tcnative is building its own .so library, and still loading tomcat's one?

--- netty-tcnative-netty-tcnative-1.1.30.Fork2/src/main/java/org/apache/tomcat/jni/Library.java
+++ netty-tcnative-netty-tcnative-1.1.30.Fork2/src/main/java/org/apache/tomcat/jni/Library.java
@@ -24,7 +24,7 @@
 public final class Library {
 
     /* Default library names */
-    private static final String [] NAMES = {"tcnative-1", "libtcnative-1"};
+    private static final String [] NAMES = {"netty-tcnative", "libnetty-tcnative", "netty-tcnative-1", "libnetty-tcnative-1"};
     /*
      * A handle to the unique Library singleton instance.
      */
```

Or remove native sources and build from the fork.

Thank you!

**Comment by normanmaurer** [comment]
@judovana did you try this patch and check if it still works ?

**Comment by judovana** [comment]
Hello. Yes. 

Longer story -  it was *not* working without  the patch (unless you installed original tomcat tcnative provider)
At the end littlebit different patch is in use - http://pkgs.fedoraproject.org/cgit/netty-tcnative.git/tree/fixLibNames.patch.in - but the names are kept.

**Comment by normanmaurer** [comment]
@judovana interesting... Could you do a PR so we can fix it upstream here ?

**Comment by judovana** [comment]
Ok. I will do my best tomorrow CET.

**Comment by normanmaurer** [comment]
@judovana awesome... please also ensure you sign our ICLA:
http://netty.io/s/icla

**Comment by judovana** [comment]
Signed and created:
https://github.com/netty/netty-tcnative/pull/25

**Comment by normanmaurer** [comment]
Fixed by #25 (3c08aed351dfa3316978dfd44b62b850ced12d14) ... Thanks again @judovana
